client/src/constants/karmaGap.ts

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+// In Karma GAP our epochs are named "Programs".
+export const PROGRAMS_IDS_TO_EPOCH_NUMBER_MAPPING = {
+  1: '544',
+  2: '545',
+  3: '546',
+  4: '547',
+  5: '548',
+};
+/* eslint-enable @typescript-eslint/naming-convention */
+
+export const API_ENDPOINT = 'https://gapapi.karmahq.xyz/';
```

Comment:
`export const KARMA_GAP_API_ENDPOINT = 'https://gapapi.karmahq.xyz/';`

Reply:
From `karmaGap` file? It seems like duplication of names.